package.json

```diff
@@ -105,7 +105,6 @@
     "netmask": "^2.0.1",
     "pubnub/superagent-proxy": "^3.0.0",
     "pull-ws": "^3.3.2",
-    "ws": "^7.4.6",
```

Comment:
I'm not sure this is necessary anymore. Was added in https://github.com/MetaMask/metamask-extension/pull/11229. Tested by removing and running `yarn audit`